sof_wrapper/audit.py

```diff
@@ -37,4 +38,13 @@ def audit_entry(message, level='info', extra=None):
             if value:
                 extra[x] = value
 
+        # echo ERRORs to current_app.logger for alerts
+        if level.lower() == 'error':
+            # remove obvious PHI
+            scrubbed_extra = deepcopy(extra)
+            for x in ('user', 'subject', 'patient'):
```

Comment:
I don't think the `user` is PHI, but don't think it'd be necessarily helpful

Reply:
user often includes their DEA, figured it was worth protecting